responses/07_update-files.md

```diff
@@ -76,12 +76,24 @@ That was a lot of information that you just learned. @{{user.login}} it is time
       </details>
 
 2. Save the changes to each file
-3. Commit the changes to this branch
-   `git add main.js action.yml workflow.yml`
-   `git commit -m 'allow input in all action files'`
-4. Push the changes from your local machine to this repository.
-   `git push`
+3. Commit the changes to this branch and push them to GitHub:
+
+   ```shell
+   git add main.js action.yml workflow.yml
```

Comment:
@hectorsector these files are in different paths so this `git add` does not work.  We also don't know which directory the user is in, because of that I'm drawing a blank on how to handle fixing this

Reply:
We can do the good ol' `git add .`

Reply:
I think we may have to in this case